flight/PiOS/STM32/pios_servo.c

```diff
@@ -248,7 +250,13 @@ void PIOS_Servo_SetMode(const uint16_t *out_rate, const int banks, const uint16_
 			const struct pios_tim_channel *chan = &servo_cfg->channels[j];
 			if (timer_banks[i].timer == chan->timer) {
 				/* save the frequency for these channels */
-				output_channel_mode[j] = (out_rate[i] == 0) ? SYNC_PWM : REGULAR_PWM;
+				if (out_rate[i] == 0) {
+					output_channel_mode[j] = SYNC_PWM;
+				} else if (channel_max[j] >= channel_min[j]) {
+					output_channel_mode[j] = REGULAR_PWM;
+				} else {
+					output_channel_mode[j] = REGULAR_INVERTED_PWM;
+				}
```

Comment:
For clarity, a real PWM channel can be dangerous to nail "active" because it might be hooked to a transistor and then a load.  For a oneshot type channel, going all-high or all-low should be safe / cause the actuator to disarm.